bin/run-in-docker.sh

```diff
@@ -1,39 +1,71 @@
 #!/usr/bin/env bash
 set -e
 
-# Synopsis:
-# Test runner for run.sh in a docker container
-# Takes the same arguments as run.sh (EXCEPT THAT SOLUTION AND OUTPUT PATH ARE RELATIVE)
-# Builds the Dockerfile
-# Runs the docker image passing along the initial arguments
+GREEN="\033[0;32m"
+ORANGE="\033[0;33m"
+NC="\033[0m"
 
-# Arguments:
-# $1: exercise slug
-# $2: **RELATIVE** path to solution folder (with trailing slash)
-# $3: **RELATIVE** path to output directory (with trailing slash)
-
-# Output:
-# Writes the test results to a results.json file in the passed-in output directory.
-# The test results are formatted according to the specifications at https://github.com/exercism/automated-tests/blob/master/docs/interface.md
-
-# Example:
-# ./run-in-docker.sh two-fer ./relative/path/to/two-fer/solution/folder/ ./relative/path/to/output/directory/
-
-# If arguments not provided, print usage and exit
-if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ]; then
-    echo "usage: run-in-docker.sh exercise-slug ./relative/path/to/solution/folder/ ./relative/path/to/output/directory/"
+printHelpAndExit() {
+    echo -e "${ORANGE}USAGE${NC}"
+    echo -e "   ./run-in-docker.sh ${GREEN}-s${NC} two-fer ${GREEN}-i${NC} /absolute/path/to/solution/folder/ ${GREEN}-o${NC} ./absolute/path/to/output/directory/"
```

Comment:
I've converted the relative paths to absolute paths as there is no guarantee that the path is relative (see https://github.com/exercism/docs/blob/main/building/tooling/test-runners/docker.md) and in fact it is more likely for it to be absolute.